### PR TITLE
Fix multiprocessing auth

### PR DESCRIPTION
**Pull request by jamesblackburn**

### CODE REVIEW

File: arctic/arctic.py [hunk 1]
```diff
@@ -144,12 +144,14 @@ def _conn(self):
             return self.__conn
 
     def reset(self):
+        logger.debug("Arctic.reset()")
```
Comment (bmoscon) [inline arctic/arctic.py]: can't tell if this is an actual issue or a github formatting issue, but this line looks to be indented incorrectly.

Comment (jamesblackburn) [reply arctic/arctic.py]: Looks ok to me...

Comment (bmoscon) [reply arctic/arctic.py]: ok, on my monitoir it had one space more than the line below it. :)

**Comment by dimosped** [comment]
there is also:
https://github.com/manahl/arctic/pull/477/files

File: arctic/arctic.py [hunk 2]
```diff
@@ -447,6 +449,7 @@ def _auth(self, database):
             authenticate(database, auth.user, auth.password)
 
     def reset_auth(self):
+        logger.debug("reset_auth() %s" % self)
```
Comment (dimosped) [inline arctic/arctic.py]: we'd better move it in _auth() 
the reset_auth is not being used (much).

Comment (dimosped) [reply arctic/arctic.py]: since we no longer spam the auth with every call, it won't add much of a perf hit

Comment (jamesblackburn) [reply arctic/arctic.py]: I'm already getting logs for calls into `_auth` as `authenticate` does a debug log on each call

**Pull request merged**